src/NearOrg/DataAvailabilityPage.jsx

```diff
@@ -0,0 +1,319 @@
+  min-height: 540px;
+  display: flex;
+  align-items: center;
+  background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAJYAAACWCAYAAAA8AXHiAAAACXBIWXMAABYlAAAWJQFJUiTwAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAGeSURBVHgB7doxTisxEAbgeY/mvQro6NiSDo6QkpJbcA2OwjWooKQMJ2DpKENJBV7FEYoBeQSIZr9PGk2cItWvsdfZnSBjKHVf6rnUbdD1N8g4K7VX6jhIEaycofaTIEWwcoam0yFYOYe179WiQ7Byhk8+8wnB6munlHNWgmD1tUGyFSYIVl8bJFcOCYLV106s/aBrJ2hNE+qo1GmpRanz2J5aB6X+x/oQv/l+FWz5E/O1iHU4pom0W/u0/uoZahnrgN2VGuv6Jpidl1+o2T5BznkrfKj9MdZT6l9836r+3k2pq1KXMVNz3gpbU7hOmj49AQ7x/lJ0WWsK5xhv2+AYkHQR29vbddDluqFvbNZPQZdg9S07az4gWH3tHZVgJQhW3xjb4XIZyo+Z3nffHN79CZ1gYuXc1b4KEytFsHLGptMhWDlj7Q9BimDlbJ4Ex4AftggHdwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIpXoUVLSWulnzoAAAAASUVORK5CYII=");
+  background-size: 75px 75px;
+  background-repeat: repeat;
+  background-position: center top;
+
+  @media (max-width: 900px) {
+    min-height: 390px;
+  }
+`;
+
+const PatternContent = styled.div`
+  padding: 1rem;
+  max-width: 808px;
+  margin: 0 auto;
+  background-color: var(--background-color);
+  display: flex;
+  align-items: center;
+  min-height: 260px;
+
+  @media (max-width: 900px) {
+    min-height: 0px;
+  }
+`;
+
+return (
+  <Wrapper>
+    <Section backgroundColor="#F2F1EA" style={{ padding: "72px 0" }}>
+      <Container center>
+        <Pattern>
+          <PatternContent>
+            <Flex gap="32px" direction="column" alignItems="center">
+              <H1>An efficient and robust data availability layer</H1>
+
+              <Text size="text-l" mobileSize="text-base">
+                Simplify and lower costs in your rollup network by using NEAR as
+                the Data Availability (DA) Layer in your Rollup Stack
+              </Text>
+            </Flex>
+          </PatternContent>
+        </Pattern>
+      </Container>
+    </Section>
+
+    <Section>
+      <Container>
+        <Flex direction="column" gap="24px">
+          <H2 style={{ maxWidth: "1016px" }}>
+            A data availability layer compatible with L2 frameworks
+          </H2>
+          <Text
+            size="text-2xl"
+            mobileSize="text-l"
+            style={{ maxWidth: "808px" }}
+          >
+            Plug NEAR’s DA layer into your L2 framework and start storing proofs
+            on a blockchain with a proven trajectory of 100% uptime over its
+            lifetime.
+          </Text>
+        </Flex>
+
+        <Widget
+          src="${REPL_ACCOUNT}/widget/NearOrg.ContentWithImage"
+          key="l2-frameworks"
+          props={{
+            content: (
+              <Flex direction="column" gap="60px">
+                <Flex direction="column" gap="24px">
+                  <Text size="text-xl" mobileSize="text-l" fontWeight="500">
+                    Drastically reduce your costs
+                  </Text>
+                  <Text>
+                    Storing calldata on NEAR Protocol is approximately 8000x
```

Comment:
@calebjacob is `calldata` intentionally one word?

Reply:
I just copied this from the text I was given. I did a quick Google and it looks like `calldata` is an actual term, haha.